backend/core/src/assets/assets.controller.ts

```diff
@@ -27,8 +38,8 @@ export class AssetsController {
     return this.assetsService.create(asset)
   }
 
-  @Put()
-  @ApiOperation({ summary: "Create asset", operationId: "create" })
+  @Put(`:assetId`)
+  @ApiOperation({ summary: "Update asset", operationId: "create" })
```

Comment:
Maybe I'm misunderstanding the [swagger document for operationId](https://swagger.io/docs/specification/paths-and-operations/), but it seems to me that it's supposed to be unique among **all** operations in the API? so this would be something like `assetsUpdateById` or something uniquely identifying like that, not `create`? (affects the reset of the decorators too, I just noticed it here)